_haikus/freshness.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: freshness
+author: tcd
+---
+
+the farm has freshness
```

Comment:
Please add a `<br>` tag or two spaces — `  ` — to break the lines on your haiku.

Reply:
break tags added!